### ISSUE TITLE
[MU4 Issue] No Split Display options?

### ISSUE THREAD

**Issue by rgreen5** [musescore/MuseScore#12248]
**Describe the issue**

In MS3 there are [Split display](https://musescore.org/en/handbook/3/viewing-and-navigation#display) options. In MS4, these are absent.

I understand that for technical reasons it is no longer possible to have multiple tabs in the same program window for multiple scores, but surely it should still be possible to retain the split display options for a _single_ score? IME, this is very useful, for example, for copying a section from one part of a score to another.

Now this will have to be done by opening another MS4 window. If this is the case, will it still be possible to cut/copy/paste from one window to the other?

**Comment by Tantacrul** [comment]
Yes. We intend to have a split display for a single score but have delayed this until 4.0 is out. Closing for now. Will reopen after launch.

**Comment by Jojo-Schmitz** [comment]
Please reopen, 4.0 is out since more than a year now...

**Comment by Tantacrul** [comment]
Done

**Comment by wizofaus** [comment]
Definitely needed - and particularly in separate windows so they can be displayed on different monitors.

**Comment by Rusca8** [comment]
Is there any plan on this? It'd be extremely useful for adding variation to reexplositions.

**Comment by tbmartinct** [comment]
Yes! Please hurry!  This is huge inconvenience when trying to copy from one part of a score to another.  It would also be useful for me to have a "helicopter view" of the whole score on one monitor and a zoomed-in view on another where I can focus on just a few staves for note input.

**Comment by SteveBlower** [comment]
That would have been better posted in Discussions on Github or on the musescore.org forums rather than adding clutter to this Github issue.  But for a "helicopter view" try the timeline available with f12 or via the view menu

**Comment by tbmartinct** [comment]
By "helicopter" I was looking to see a zoomed out view of the actual score so I get a sense of which instruments are playing. That's not what the timeline shows, and the Navigator is so small that it can't be used for that purpose either. 

And of course, this comment doesn't belong here either. Maybe it's just me, but each time I try to engage I'm doing it in the "wrong place." I posted it in a forum thread on this topic, but the most recent post before mine was from June of 2024. The most recent before my post here was July 2025. This is major productivity hinderance for me, and the issue seems to be dormant.

**Comment by MarcSabatella** [comment]
The timeline does show you which instruments are playing when - perhaps you haven't scrolled down to the place where that happens (below the part showing tempo changes, rehearsal marks, etc)?

As for where to post: if you have new technical information that would help solve an issue, then adding a comment to it is appropriate.  If you just want to to regiaster your support for an idea, a "thumbs up" icon on the main post is sufficient.  If you would like to discuss an issue in other ways, the forums are the appropriate place for that.  The issue itself is meant to be focused on *just* the description of how to reproduce the issue and possibly technical insights into solving it.

**Comment by Jojo-Schmitz** [comment]
Came up again in https://musescore.org/en/node/390221
